src/node/stage/approved.rs

```diff
@@ -1136,23 +1354,52 @@ impl Approved {
         self.send_routing_message(core, src, dst, content, Some(knowledge_index))
     }
 
+    fn notify_sibling(
```

Comment:
It seems this method is not only for siblings, but also for any nodes that are currently elders but are about to get demoted, is it? If so, then maybe it should be renamed to reflect it. That probably applied to the `DKGSibling` variant. Not sure what a better name should be though...

Reply:
Yes you are right.
However I think the current name reflects the `purpose` and the case you described is a un-necessary messaging which will be ignored.
I'd prefer remain with this naming.

Reply:
Yes but sibling is like children of the Elders really and they are not. They are perhaps `NonVoting` nodes?

Reply:
Right. If the purpose of this is to notify siblings, then it's fine. But I wonder, isn't this used also in the non-split case to notify nodes that are currently elders, but won't be in the next one?

Reply:
They are just `old elders` that not vote for this because got split into the other side during split.

How about change it to NotifyOldElders ? and same to the variant?

Reply:
> isn't this used also in the non-split case to notify nodes that are currently elders, but won't be in the next one?

yes. the message will be sent. However it shall be ignored on the receiver side.

Reply:
I feel we are playing with this a bit. A node either has voting rights in a section or not, so old_elders would be just non voting nodes (no longer elders) in this section. I think it would be best to stick with Elder as meaning voting node. So notifying new section elders perhaps should be OK as long as we identify the correct voting group and that is implicit really if the section key is used as an index and more in a split as the prefix has got longer.

Reply:
> yes. the message will be sent. However it shall be ignored on the receiver side.

Ah OK. This is what I was unsure about. So this is only meant for the split case then, right?

Reply:
> So this is only meant for the split case then, right?

yes